docs/en/sql-reference/functions/array-functions.md

```diff
@@ -176,6 +176,52 @@ hasAny(array1, array2)
 
 `SELECT hasAll([[1, 2], [3, 4]], [[1, 2], [1, 2]])` returns `1`.
 
+## hasSubStr {#hassubseq}
+
+Checks whether all the elements of array2 appear in array1 in the same exact order.
+
+``` sql
+hasSubStr(array1, array2)
+```
+
+In other words, the functions will check whether all the elements of `array2` are contained in `array1` like 
+the `hasAll` function. In addition, it will check that the elements are observed in the same order in both `array1` and `array2`.
+
+For Example, `hasSubStr([1,2,3,4], [2,3])` will return 1. However, `hasSubStr([1,2,3,4], [3,2])` will return `0`.
```

Comment:
To make it more intuitive adding another example here might help, I guess. Something like "... hasSubStr([1,2,3,4], [1,2,3]) returns 1, and hasSubStr([1,2,3,4], [1,2,4]) returns 0 ..."